packages/abi-ts/src/debug.ts

```diff
@@ -0,0 +1,3 @@
+import createDebug from "debug";
+
+export const debug = createDebug("abi-ts");
```

Comment:
intentionally didn't prefix with `mud:` because this package doesn't have any MUD dependencies and kind of lives outside the framework